db/migrations/2021_12_30_000000_remove_admin_news_html_privilege.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
This should also be added to `tests/Unit/HasDatabase.php` as its not needed to run on unit tests